packages/twenty-server/src/engine/core-modules/auth/services/sign-in-up.service.ts

```diff
@@ -181,11 +181,13 @@ export class SignInUpService {
       value: true,
     });
 
-    await this.onboardingService.setOnboardingCreateProfileCompletion({
-      userId: user.id,
-      workspaceId: workspace.id,
-      value: true,
-    });
+    if (firstName === '' && lastName === '') {
```

Comment:
I don't think this is actually right anymore, we should always show the create profile popin, it is prefill with currentInformation